### ISSUE TITLE
Modify (or remove) 'gl' fallback command

### ISSUE THREAD

**Issue by salcode** [salcode/salcode-zsh#16]
Currently we are defining `gl` with

```
alias gl="git lg || git log --oneline --graph"
```

however the problem is if we define a specific branch.

e.g. when we are `main` we can do

```
git lg develop  # this works
```

however if we do

```
gl develop # this does NOT work
```

I think this is not working because of the fallback code we have in our alias.

I suspect there is a way we can make this work with the fallback _and_ accept parameters, however it is probably easier to just drop the fallback command.

**Comment by salcode** [comment]
Removed in #18

**Comment by salcode** [comment]
Rather than removing the `gl` fallback completely, I'd like to find a way to handle this more elegantly.

e.g.

Replacing this alias with a function, e.g. 

```
function gl() {
	git lg "$@" || git log --oneline "$@"
}
```

is better but does have the disadvantage that if `git lg` is not defined, we can an error message before the `git log --oneline` output

```
git: 'lg' is not a git command. See 'git --help'.

The most similar command is
        log
```

however, even with this disadvantage - this is better.

**Comment by salcode** [comment]
Ideally, it seems we could store the standard output in a variable when we run `git lg` and then check the standard error

if there was not error, we output the standard output from our variable and stop

if there was an error, we run `git log --oneline --graph "$@"` instead (and never output the original standard output)

**Comment by salcode** [comment]
I think this will work

```
function gl() {
	OUTPUT=$(git lg "$@" 2>/dev/null)
	if [ $? -eq 0 ];
	then
		echo $OUTPUT | $(git config core.pager) -R;
	else
		git log --oneline --graph "$@"
	fi
}
```

**Comment by salcode** [comment]
I think this will also work and is significantly simpler

```
function gl() {
	git lg "$@" 2>/dev/null || git log --oneline --graph "$@"
}
```

**Comment by salcode** [comment]
Resolved in d6874406cd3e939861e5ee26d8f5c0276254cee3

See https://salferrarello.com/zsh-alias-with-fallback/ for details explanation
